src/libraries/System.Threading.Timer/tests/TimerChangeTests.cs

```diff
@@ -38,14 +38,14 @@ public void Timer_Change_Period_OutOfRange_Throws()
         }
 
         [Fact]
-        public void Timer_Change_AfterDispose_Throws()
+        public void Timer_Change_AfterDispose_Test()
         {
             var t = new Timer(new TimerCallback(EmptyTimerTarget), null, 1, 1);
             t.Dispose();
-            Assert.Throws<ObjectDisposedException>(() => t.Change(1, 1));
-            Assert.Throws<ObjectDisposedException>(() => t.Change(1L, 1L));
-            Assert.Throws<ObjectDisposedException>(() => t.Change(1u, 1u));
-            Assert.Throws<ObjectDisposedException>(() => t.Change(TimeSpan.FromMilliseconds(1), TimeSpan.FromMilliseconds(1)));
+            Assert.False(t.Change(1, 1));
```

Comment:
Do we have tests elsewhere the validate it correctly returns true prior to Dispose?